tutorials/preprocessing/plot_15_handling_bad_channels.py

```diff
@@ -0,0 +1,209 @@
+# gets updated immediately each time you toggle a channel, and will retain its
+# modified state after the plot window is closed.
+#
+# The list of bad channels in the :class:`mne.Info` object's ``bads`` field is
+# automatically taken into account in dozens of functions and methods across
+# the MNE-Python codebase. This is done consistently with a parameter
+# ``exclude='bads'`` in the function or method signature. Typically this
+# ``exclude`` parameter also accepts a list of channel names or indices, so if
+# you want to *include* the bad channels you can do so by passing
+# ``exclude=[]`` (or some other list of channels to exclude).
+#
+#
+# When to look for bad channels
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# You can start looking for bad channels during the experiment session when the
+# data is being acquired. If you notice any flat or excessively noisy channels,
+# you can note them in your experiment log or protocol sheet. If your system
+# computes on-line averages, these can be a good way to spot bad channels as
+# well. After the data has been collected, you can do a more thorough check for
+# bad channels by browsing the raw data using :meth:`mne.io.Raw.plot`, with any
+# projectors or ICA applied. Finally, you can compute off-line averages (again
+# with projectors, ICA, and EEG referencing disabled) to look for channels with
+# unusual properties.
+#
+# Remember, marking bad channels should be done as early as possible in the
+# analysis pipeline. When bad channels are marked in a :class:`~mne.io.Raw`
+# object, the markings will be automatically transferred through the chain of
+# derived object types: including :class:`~mne.Epochs` and :class:`~mne.Evoked`
+# objects, but also :class:`noise covariance <mne.Covariance>` objects,
+# :class:`forward solution computations <mne.Forward>`, and
+# :class:`inverse operators <mne.minimum_norm.InverseOperator>`.
+#
+#
+# Why mark bad channels at all?
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Many analysis computations can be strongly affected by the presence of bad
+# channels. For example, a malfunctioning channel with completely flat signal
+# will have zero channel variance, which will cause noise estimates to be
+# unrealistically low. This low noise estimate will lead to a strong channel
+# weight in the estimate of cortical current, and because the channel is flat,
+# the magnitude of cortical current estimates will shrink dramatically.
+#
+# Conversely, very noisy channels can also cause problems. For example, they
+# can lead to too many epochs being discarded based on signal amplitude
+# rejection thresholds, which in turn can lead to less robust estimation of the
+# noise covariance across sensors. Noisy channels can also interfere with
+# :term:`SSP <projector>` computations, because the projectors will be
+# spatially biased in the direction of the noisy channel, which can cause
+# adjacent good channels to be suppressed.
+#
```

Comment:
We could add: ... on the other hand, when performing machine learning analyses bad channels may have limited, if any impact.